test/conformance/exp_command_buffer/update/buffer_fill_kernel_update.cpp

```diff
@@ -336,7 +336,7 @@ TEST_P(BufferFillCommandTest, OverrideUpdate) {
     ASSERT_SUCCESS(urCommandBufferUpdateKernelLaunchExp(command_handle,
                                                         &first_update_desc));
 
-    uint32_t second_val = -99;
```

Comment:
I think the usage of a negative value here and in the next test was an oversight, and using `0xffffff9d; // -99` is just making the test less readable than a simple unsigned value. Could we change the `-99` to `99` (or any unsigned value other than `33` from `first_val` really)

Reply:
These are both now using `99`.